assets/css/_route_picker.scss

```diff
@@ -0,0 +1,52 @@
+.m-route-picker {
+  background-color: $light-periwinkle;
+  height: 100%;
+  overflow-y: scroll;
+  padding: 1rem;
+  position: fixed;
+  width: 7.5rem;
+  z-index: 2;
+}
+
+.m-route-picker__selected-routes,
+.m-route-picker__route-list {
+  list-style-type: none;
+  padding: 0;
+}
+
+.m-route-picker__selected-routes {
+  display: flex;
+  flex-wrap: wrap;
+  margin-bottom: 2rem;
+  margin-top: 0;
+  min-height: 2rem;
+}
+
+.m-route-picker__selected-routes-button,
+.m-route-picker__route-list-button {
+  border: none;
+  border-radius: 4px;
+  color: $dark;
+  font-family: $font-family;
+  font-size: 0.8125rem;
+  font-weight: bold;
+  padding: 0.3125rem;
+}
+
+.m-route-picker__selected-routes-button {
+  background-color: $sunflower;
+  margin-bottom: 0.25rem;
+  margin-right: 0.25rem;
+  min-width: 2rem;
+}
+
+.m-route-picker__route-list-button {
+  background-color: transparentize($color: $white, $amount: 0.5);
```

Comment:
This requires `...--selected` buttons to override `background-color`. I personally like to avoid overriding styles. Thoughts on only applying this property to `...--unselected` elements? I guess here the overriding style is so close that I'm not sure if that'd be any clearer in this case.

Reply:
I removed the `--deselected` version.